int32 monitoring_case_number
int32[] fields
bool[] fields_valid
